Pipelines/Templates/set-website-variable.yml

```diff
@@ -0,0 +1,38 @@
+#We are going to set websiteId as a pipeline variable only if there is a portal in the Dataverse
+
+parameters:
+- name: serviceConnectionUrl
+  type: string
+  #In case of exporting portal, we expect that the solution unique name will be the same as the portal name
+- name: solutionName
+  type: string
+
+steps:
+- powershell: | 
+    $websiteId = ${env:WEBSITEID}
+    if ($websiteId -ne $null -and $websiteId -ne '') {
+      #WebsiteId is set in the pipeline variable. We don't need to do anything.
+      exit 0
+    }
+
+    # load PowerShell files into memory
+    . "$env:POWERSHELLPATH/dataverse-webapi-functions.ps1"
+    $dataverseHost = Get-HostFromUrl "${{parameters.serviceConnectionUrl}}"
+    $odataQuery = "adx_websites?`$filter=adx_name eq '${{parameters.solutionName}}'"    
+
+    try{
+      $response = Invoke-DataverseHttpGet $env:SPNTOKEN $dataverseHost $odataQuery
```

Comment:
@mawasile I don't think SPNTOKEN variable will be available in this context. We might need to invoke Get-SpnToken() function. FYI @mikefactorial